OpenStack-MonitoringScripts/usr/local/bin/limits-to-influx.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python
+import json
+import sys
+import os
+import requests
+from subprocess import Popen, PIPE
+from ConfigParser import SafeConfigParser
+
+env = os.environ.copy()
+
+# Read from config file
+parser = SafeConfigParser()
+try:
+   parser.read('/etc/influxdb.conf')
+   host = parser.get('db', 'host')
+   database = parser.get('db', 'database')
+   username = parser.get('auth', 'username')
+   password = parser.get('auth', 'password')
+   instance = parser.get('cloud','instance')
+except:
+   print 'Unable to read from config file'
+   sys.exit(1)
+
+url = 'http://'+host+'/write?db='+database +'&precision=s'
```

Comment:
Can we use f string here:

`f"http://{host}'/write?db='{database}'&precision=s